Dockerfile

```diff
@@ -15,5 +15,9 @@ RUN curl -Lfs "https://github.com/openshift/rosa/releases/download/$(curl -Lfs h
 RUN curl -LSs -o /usr/local/bin/opm "https://github.com/operator-framework/operator-registry/releases/download/$(curl -Lfs https://api.github.com/repos/operator-framework/operator-registry/releases/latest \
     | jq -r .tag_name)/linux-amd64-opm" \
     && chmod 0755 /usr/local/bin/opm
+RUN curl -LSs -o awscli.zip "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" \
```

Comment:
Maybe would be better to install it with pip? `pip install awscli` what you think?

Reply:
Neither `python` no `pip` is installed so using `pip` won't be any simpler than this imo. Whether it is better I am not sure, I would say it does not matter much. I am not familiar enough with `pip install` to say whether the cleanup (`rm`) will or will not be required if it is used or whether the resulting image will be smaller.